src/test/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayoutTest.java

```diff
@@ -134,29 +135,33 @@ public void testNoThreadCustomDateFormatOutput() throws JoranException, IOExcept
 
     @Test
     public void testOutputWhenAlertLevelIsDisabled() throws JoranException, IOException {
-        configLogback(LOGBACK_WITHOUT_ALERT_LEVEL);
+        System.setProperty("LOGBACK_REQUIRE_ALERT_LEVEL", "false");
+        configLogback(LOGBACK);
 
         log.error("message", new DummyP3Exception("oh no"));
 
         String timestamp = "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}(\\+|-)\\d{4}";
+        String thread = "\\[" + Thread.currentThread().getName() + "\\] ";
         String logger = this.getClass().getCanonicalName();
 
         assertThat(baos.toString()).containsPattern(
-            timestamp + " ERROR " + logger + ":\\d+: 0. message\n"
+            timestamp + " ERROR " + thread + logger + ":\\d+: 0. message\n"
         );
     }
 
     @Test
     public void testOutputWhenErrorCodeIsDisabled() throws JoranException, IOException {
-        configLogback(LOGBACK_WITHOUT_ERROR_CODE);
+        System.setProperty("LOGBACK_REQUIRE_ERROR_CODE", "false");
+        configLogback(LOGBACK);
 
         log.error("message", new DummyP3Exception("oh no"));
 
         String timestamp = "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}(\\+|-)\\d{4}";
+        String thread = "\\[" + Thread.currentThread().getName() + "\\] ";
```

Comment:
was this not logged before? if it was, did tests pass previously?

Reply:
These tests use logback configuration from source code instead. just a matter of system environments to test output format